docs/changelog.md

```diff
@@ -1,4 +1,7 @@
 # TBD
+### Breaking Changes
```

Comment:
Breaking changes lets users know about the changes part - let's also add a section to `Features` to let them know that there's new functionality available!